xen-tools: Disable installation of the python3-core inside initrd

svlad-90 opened the pull request:
After switching from the Dunfel to the Kirkstone release of Yocto the
size of the Dom0 initramfs was drastically increased.

This commit is another step in direction of the initramfs size's
minimization.

The runtime dependency to the python3-core looks redundant. Thus, we
remove it.

Signed-off-by: Vladyslav Goncharuk <vladyslav_goncharuk@epam.com>

----

prod-devel-rcar.yaml: Remove the custom INITRAMFS_MAXSIZE parameter

Usage of this parameter is not needed anymore, after the size of the
initramfs was reduced.

Signed-off-by: Vladyslav Goncharuk <vladyslav_goncharuk@epam.com>

Commit:
xen-tools: Disable installation of the python3-core inside initrd

After switching from the Dunfel to the Kirkstone release of Yocto the
size of the Dom0 initramfs was drastically increased.

This commit is another step in direction of the initramfs size's
minimization.

The runtime dependency to the python3-core looks redundant. Thus, we
remove it.

Signed-off-by: Vladyslav Goncharuk <vladyslav_goncharuk@epam.com>

svlad-90 commented:
The size of initrd was reduced

![Screenshot from 2023-02-09 16-23-53](https://user-images.githubusercontent.com/4373469/217839815-191663aa-31cb-478e-a8b1-ad4cbe965f68.png)

Initramfs now does not contain any "*.py" or "*.python" files.

**Before:**

```
vladyslav_goncharuk@EPUAODEW00C5:/mnt/moulin/deploy/prod-devel-rcar/my_devel_kirkstone/tmp/core-image-thin-initramfs-generic-armv8-xt-20230207125919.rootfs$ find . -type f -name "*python*"
./usr/lib/python3.10/__pycache__/sre_compile.cpython-310.pyc
./usr/lib/python3.10/__pycache__/argparse.cpython-310.pyc
./usr/lib/python3.10/__pycache__/os.cpython-310.pyc
./usr/lib/python3.10/__pycache__/runpy.cpython-310.pyc
./usr/lib/python3.10/__pycache__/copyreg.cpython-310.pyc
./usr/lib/python3.10/__pycache__/pathlib.cpython-310.pyc
./usr/lib/python3.10/__pycache__/genericpath.cpython-310.pyc
./usr/lib/python3.10/__pycache__/code.cpython-310.pyc
./usr/lib/python3.10/__pycache__/weakref.cpython-310.pyc
./usr/lib/python3.10/__pycache__/site.cpython-310.pyc
./usr/lib/python3.10/__pycache__/tokenize.cpython-310.pyc
./usr/lib/python3.10/__pycache__/linecache.cpython-310.pyc
./usr/lib/python3.10/__pycache__/_sysconfigdata__linux_aarch64-linux-gnu.cpython-310.opt-2.pyc
./usr/lib/python3.10/__pycache__/codeop.cpython-310.pyc
./usr/lib/python3.10/__pycache__/gettext.cpython-310.pyc
./usr/lib/python3.10/__pycache__/lzma.cpython-310.pyc
./usr/lib/python3.10/__pycache__/configparser.cpython-310.pyc
./usr/lib/python3.10/__pycache__/subprocess.cpython-310.pyc
./usr/lib/python3.10/__pycache__/selectors.cpython-310.pyc
./usr/lib/python3.10/__pycache__/ast.cpython-310.pyc
./usr/lib/python3.10/__pycache__/threading.cpython-310.pyc
./usr/lib/python3.10/__pycache__/getopt.cpython-310.pyc
./usr/lib/python3.10/__pycache__/token.cpython-310.pyc
./usr/lib/python3.10/__pycache__/re.cpython-310.pyc
./usr/lib/python3.10/__pycache__/_sysconfigdata__linux_aarch64-linux-gnu.cpython-310.pyc
./usr/lib/python3.10/__pycache__/_weakrefset.cpython-310.pyc
./usr/lib/python3.10/__pycache__/rlcompleter.cpython-310.pyc
./usr/lib/python3.10/__pycache__/stat.cpython-310.pyc
./usr/lib/python3.10/__pycache__/copy.cpython-310.pyc
./usr/lib/python3.10/__pycache__/abc.cpython-310.pyc
./usr/lib/python3.10/__pycache__/shutil.cpython-310.pyc
./usr/lib/python3.10/__pycache__/enum.cpython-310.pyc
./usr/lib/python3.10/__pycache__/functools.cpython-310.pyc
./usr/lib/python3.10/__pycache__/optparse.cpython-310.pyc
./usr/lib/python3.10/__pycache__/operator.cpython-310.pyc
./usr/lib/python3.10/__pycache__/types.cpython-310.pyc
./usr/lib/python3.10/__pycache__/platform.cpython-310.pyc
./usr/lib/python3.10/__pycache__/keyword.cpython-310.pyc
./usr/lib/python3.10/__pycache__/ntpath.cpython-310.pyc
./usr/lib/python3.10/__pycache__/dis.cpython-310.pyc
./usr/lib/python3.10/__pycache__/struct.cpython-310.pyc
./usr/lib/python3.10/__pycache__/pkgutil.cpython-310.pyc
./usr/lib/python3.10/__pycache__/contextlib.cpython-310.pyc
./usr/lib/python3.10/__pycache__/posixpath.cpython-310.pyc
./usr/lib/python3.10/__pycache__/_compression.cpython-310.pyc
./usr/lib/python3.10/__pycache__/textwrap.cpython-310.pyc
./usr/lib/python3.10/__pycache__/io.cpython-310.pyc
./usr/lib/python3.10/__pycache__/opcode.cpython-310.pyc
./usr/lib/python3.10/__pycache__/typing.cpython-310.pyc
./usr/lib/python3.10/__pycache__/reprlib.cpython-310.pyc
./usr/lib/python3.10/__pycache__/fnmatch.cpython-310.pyc
./usr/lib/python3.10/__pycache__/stringprep.cpython-310.pyc
./usr/lib/python3.10/__pycache__/sre_constants.cpython-310.pyc
./usr/lib/python3.10/__pycache__/inspect.cpython-310.pyc
./usr/lib/python3.10/__pycache__/warnings.cpython-310.pyc
./usr/lib/python3.10/__pycache__/imp.cpython-310.pyc
./usr/lib/python3.10/__pycache__/sre_parse.cpython-310.pyc
./usr/lib/python3.10/__pycache__/bz2.cpython-310.pyc
./usr/lib/python3.10/__pycache__/_markupbase.cpython-310.pyc
./usr/lib/python3.10/__pycache__/codecs.cpython-310.pyc
./usr/lib/python3.10/__pycache__/_sitebuiltins.cpython-310.pyc
./usr/lib/python3.10/__pycache__/_sysconfigdata__linux_aarch64-linux-gnu.cpython-310.opt-1.pyc
./usr/lib/python3.10/__pycache__/heapq.cpython-310.pyc
./usr/lib/python3.10/__pycache__/__future__.cpython-310.pyc
./usr/lib/python3.10/__pycache__/csv.cpython-310.pyc
./usr/lib/python3.10/__pycache__/_collections_abc.cpython-310.pyc
./usr/lib/python3.10/__pycache__/sysconfig.cpython-310.pyc
./usr/lib/python3.10/__pycache__/signal.cpython-310.pyc
./usr/lib/python3.10/__pycache__/locale.cpython-310.pyc
./usr/lib/python3.10/__pycache__/bisect.cpython-310.pyc
./usr/lib/python3.10/importlib/__pycache__/_bootstrap_external.cpython-310.pyc
./usr/lib/python3.10/importlib/__pycache__/_abc.cpython-310.pyc
./usr/lib/python3.10/importlib/__pycache__/util.cpython-310.pyc
./usr/lib/python3.10/importlib/__pycache__/_adapters.cpython-310.pyc
./usr/lib/python3.10/importlib/__pycache__/_bootstrap.cpython-310.pyc
./usr/lib/python3.10/importlib/__pycache__/abc.cpython-310.pyc
./usr/lib/python3.10/importlib/__pycache__/machinery.cpython-310.pyc
./usr/lib/python3.10/importlib/__pycache__/_common.cpython-310.pyc
./usr/lib/python3.10/importlib/__pycache__/__init__.cpython-310.pyc
./usr/lib/python3.10/importlib/__pycache__/readers.cpython-310.pyc
./usr/lib/python3.10/importlib/__pycache__/resources.cpython-310.pyc
./usr/lib/python3.10/importlib/metadata/__pycache__/_collections.cpython-310.pyc
./usr/lib/python3.10/importlib/metadata/__pycache__/_adapters.cpython-310.pyc
./usr/lib/python3.10/importlib/metadata/__pycache__/_meta.cpython-310.pyc
./usr/lib/python3.10/importlib/metadata/__pycache__/_text.cpython-310.pyc
./usr/lib/python3.10/importlib/metadata/__pycache__/_itertools.cpython-310.pyc
./usr/lib/python3.10/importlib/metadata/__pycache__/__init__.cpython-310.pyc
./usr/lib/python3.10/importlib/metadata/__pycache__/_functools.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/cp855.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/iso2022_jp_1.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/cp1125.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/mac_latin2.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/euc_jis_2004.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/mac_croatian.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/cp932.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/cp437.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/cp850.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/cp720.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/cp863.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/cp950.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/big5.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/raw_unicode_escape.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/cp737.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/big5hkscs.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/gb2312.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/mac_romanian.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/cp1254.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/iso8859_15.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/iso8859_10.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/oem.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/zlib_codec.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/iso8859_5.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/cp1255.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/utf_16.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/johab.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/mac_cyrillic.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/shift_jis.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/gbk.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/unicode_escape.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/rot_13.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/iso8859_16.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/cp1257.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/euc_jisx0213.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/koi8_t.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/cp875.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/utf_32_be.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/cp861.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/mac_arabic.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/bz2_codec.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/cp1252.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/iso8859_14.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/latin_1.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/euc_jp.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/iso2022_jp_ext.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/cp852.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/punycode.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/cp864.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/mbcs.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/charmap.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/utf_7.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/iso2022_jp_3.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/koi8_r.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/iso2022_jp_2.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/utf_16_le.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/iso8859_3.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/cp866.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/cp1258.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/shift_jisx0213.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/mac_farsi.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/shift_jis_2004.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/iso8859_1.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/cp775.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/iso8859_8.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/aliases.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/euc_kr.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/cp874.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/idna.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/iso8859_11.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/iso2022_jp_2004.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/utf_8.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/cp037.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/cp1251.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/iso8859_13.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/undefined.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/__init__.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/base64_codec.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/tis_620.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/utf_32_le.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/iso2022_jp.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/cp500.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/uu_codec.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/cp1250.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/iso8859_6.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/cp858.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/utf_8_sig.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/cp1006.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/iso8859_9.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/cp865.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/koi8_u.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/hz.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/utf_32.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/cp273.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/cp860.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/quopri_codec.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/cp862.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/iso8859_7.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/cp857.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/cp1256.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/hex_codec.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/ascii.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/kz1048.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/iso8859_2.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/cp869.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/cp1026.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/cp1253.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/mac_turkish.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/mac_roman.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/gb18030.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/mac_greek.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/iso8859_4.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/cp856.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/palmos.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/cp424.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/cp1140.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/ptcp154.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/cp949.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/utf_16_be.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/iso2022_kr.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/hp_roman8.cpython-310.pyc
./usr/lib/python3.10/encodings/__pycache__/mac_iceland.cpython-310.pyc
./usr/lib/python3.10/collections/__pycache__/abc.cpython-310.pyc
./usr/lib/python3.10/collections/__pycache__/__init__.cpython-310.pyc
./usr/lib/python3.10/config-3.10-aarch64-linux-gnu/python.o
./usr/lib/python3.10/config-3.10-aarch64-linux-gnu/__pycache__/python-config.cpython-310.pyc
./usr/lib/python3.10/config-3.10-aarch64-linux-gnu/python-config.py
./usr/lib/python3.10/urllib/__pycache__/robotparser.cpython-310.pyc
./usr/lib/python3.10/urllib/__pycache__/request.cpython-310.pyc
./usr/lib/python3.10/urllib/__pycache__/response.cpython-310.pyc
./usr/lib/python3.10/urllib/__pycache__/__init__.cpython-310.pyc
./usr/lib/python3.10/urllib/__pycache__/parse.cpython-310.pyc
./usr/lib/python3.10/urllib/__pycache__/error.cpython-310.pyc
./usr/lib/python3.10/lib-dynload/_heapq.cpython-310-aarch64-linux-gnu.so
./usr/lib/python3.10/lib-dynload/_struct.cpython-310-aarch64-linux-gnu.so
./usr/lib/python3.10/lib-dynload/select.cpython-310-aarch64-linux-gnu.so
./usr/lib/python3.10/lib-dynload/_lzma.cpython-310-aarch64-linux-gnu.so
./usr/lib/python3.10/lib-dynload/binascii.cpython-310-aarch64-linux-gnu.so
./usr/lib/python3.10/lib-dynload/_csv.cpython-310-aarch64-linux-gnu.so
./usr/lib/python3.10/lib-dynload/_opcode.cpython-310-aarch64-linux-gnu.so
./usr/lib/python3.10/lib-dynload/readline.cpython-310-aarch64-linux-gnu.so
./usr/lib/python3.10/lib-dynload/_bisect.cpython-310-aarch64-linux-gnu.so
./usr/lib/python3.10/lib-dynload/array.cpython-310-aarch64-linux-gnu.so
./usr/lib/python3.10/lib-dynload/unicodedata.cpython-310-aarch64-linux-gnu.so
./usr/lib/python3.10/lib-dynload/_bz2.cpython-310-aarch64-linux-gnu.so
./usr/lib/python3.10/lib-dynload/_posixsubprocess.cpython-310-aarch64-linux-gnu.so
./usr/lib/python3.10/lib-dynload/math.cpython-310-aarch64-linux-gnu.so
./usr/lib/python3.10/lib-dynload/grp.cpython-310-aarch64-linux-gnu.so
./usr/lib/python3.10/lib-dynload/zlib.cpython-310-aarch64-linux-gnu.so
./usr/lib/libpython3.10.so.1.0
./usr/bin/python3.10
```

**After:**

```
vladyslav_goncharuk@EPUAODEW00C5:/mnt/moulin/deploy/prod-devel-rcar/my_devel_kirkstone/tmp/core-image-thin-initramfs-generic-armv8-xt-20230209130524.rootfs$ find . -type f -name "*python*"
vladyslav_goncharuk@EPUAODEW00C5:/mnt/moulin/deploy/prod-devel-rcar/my_devel_kirkstone/tmp/core-image-thin-initramfs-generic-armv8-xt-20230209130524.rootfs$
```

The line which is causing a dependency:
https://github.com/yoctoproject/poky/blob/kirkstone/meta/classes/setuptools3-base.bbclass#L3

It seems, that the runtime dependency of the class-target to the python3-core, which is declared by the setuptools3-base bbclass is redundant. The system starts successfully without it.

**=> Thus, we can remove it in order to minimize the size of the dom0's initramfs.**

otyshchenko1 commented:
Perfect, thanks! One question remains, so this limit here https://github.com/otyshchenko1/meta-xt-prod-devel-rcar/commit/f5acfef40933e2b74d2bcf240c0f4ceff6cbad82#diff-e78b63e574102e5f392ff67a928a6d7c926768b1cb99351fcf09fe007cb02501R176 is not actual and can be dropped? Could you re-check (and drop?) it?

otyshchenko1 commented:
*And* if this works well, the dom0 memory size could be reduced to it's previous size 256MB here https://github.com/otyshchenko1/meta-xt-rcar/commit/55341a375d5c8ede681d2e6957554f1c8169d830#diff-a4422f04d53193d309b47a945fa7c76fcfb1c6fa040a097a0868827e2d4dd58fR14 Could you re-check (and reduce) it?

Commit:
prod-devel-rcar.yaml: Remove the custom INITRAMFS_MAXSIZE parameter

Usage of this parameter is not needed anymore, after the size of the
initramfs was reduced.

Signed-off-by: Vladyslav Goncharuk <vladyslav_goncharuk@epam.com>

svlad-90 commented:
Hi @otyshchenko1, the INITRAMFS_MAXSIZE was removed. The image was successfully built with the default value.
